data/bss-factory-sets.json

```diff
@@ -0,0 +1,5689 @@
+          ["encore", "sacredsword"]
+        ],
+        "item": ["focussash"],
+        "nature": "jolly",
+        "evs": [0, 252, 0, 0, 4, 252],
+        "teratype": ["water", "dark", "ghost"],
+        "ability": "sharpness"
+      },
+      {
+        "species": "samurotthisui",
+        "weight": 30,
+        "moves": [
+          ["ceaselessedge"],
+          ["razorshell", "aquacutter"],
+          ["suckerpunch", "aquajet"],
+          ["sacredsword", "terablast"]
+        ],
+        "item": ["assaultvest"],
+        "nature": "jolly",
+        "evs": [0, 252, 0, 0, 4, 252],
+        "teratype": ["water", "fairy"],
+        "ability": "sharpness"
+      }
+    ]
+  },
+  "slowkinggalar": {
+    "weight": 1,
+    "sets": [
+      {
+        "species": "slowkinggalar",
+        "weight": 60,
+        "moves": [
+          ["flamethrower"],
+          ["grassknot"],
+          ["psychic", "eeriespell"],
+          ["sludgebomb"]
+        ],
+        "item": ["assaultvest"],
+        "ivs": [31, 0, 31, 31, 31, 31],
+        "nature": "modest",
+        "evs": [252, 0, 0, 252, 4, 0],
+        "teratype": ["poison", "normal"],
+        "ability": "regenerator"
+      },
+      {
+        "species": "slowkinggalar",
+        "weight": 40,
+        "moves": [
+          ["eeriespell", "sludgebomb"],
+          ["toxic", "yawn"],
+          ["trickroom", "slackoff"],
+          ["chillyreception"]
+        ],
+        "item": ["blacksludge"],
+        "ivs": [31, 0, 31, 31, 31, 31],
+        "nature": "relaxed",
+        "evs": [244, 0, 252, 0, 12, 0],
+        "teratype": ["poison"],
+        "ability": "regenerator"
+      }
+    ]
+  },
+  "basculegionf": {
+    "weight": 1,
+    "sets": [
+      {
+        "species": "basculegionf",
+        "weight": 70,
+        "moves": [
+          ["shadowball"],
+          ["aquajet"],
+          ["surf", "hydropump"],
+          ["endeavor"]
+        ],
+        "gender": "f",
+        "item": ["focussash"],
+        "nature": "rash",
+        "evs": [0, 4, 0, 252, 0, 252],
+        "teratype": ["water"],
+        "ability": "adaptability"
+      },
+      {
+        "species": "basculegionf",
+        "weight": 30,
+        "moves": [
+          ["shadowball"],
+          ["aquajet"],
+          ["surf", "hydropump"],
+          ["endeavor", "terablast"]
+        ],
+        "gender": "f",
+        "item": ["focussash"],
+        "nature": "rash",
+        "evs": [0, 4, 0, 252, 0, 252],
+        "teratype": ["fairy"],
+        "wantsTera": true,
+        "ability": "adaptability"
+      }
+    ]
+  },
+  "irontreads": {
+    "weight": 1,
+    "sets": [
+      {
+        "species": "irontreads",
+        "weight": 15,
+        "moves": [
+          ["earthquake"],
+          ["ironhead", "heavyslam"],
+          ["rapidspin", "icespinner", "voltswitch"],
+          ["knockoff"]
+        ],
+        "item": ["assaultvest"],
+        "nature": "jolly",
+        "evs": [252, 4, 4, 0, 244, 4],
```

Comment:
Jolly Breloom does not exist in this format. This spread is useless unless you have Rapid Spin, which is a 1/3 chance. Just run max/max Careful with 4 attack or something. A different spread.